package.json

```diff
@@ -24,15 +24,18 @@
         "args-parser": "^1.3.0",
         "axios": "^0.21.1",
         "bcrypt": "^5.0.1",
+        "bcryptjs": "^2.4.3",
         "bootstrap": "^5.0.2",
         "command-exists": "^1.2.9",
         "dayjs": "^1.10.6",
         "express": "^4.17.1",
         "form-data": "^4.0.0",
         "http-graceful-shutdown": "^3.1.2",
         "jsonwebtoken": "^8.5.1",
+        "node-gyp": "^3.8.0",
```

Comment:
What is the purpose? If it is for native build, since non-Docker users are likely getting error during "npm install", it is hard to install.

Please check:
https://github.com/louislam/uptime-kuma/wiki/%5BDev%5D-Setup-Development-Environment

Reply:
Thanks, I'll sort that now, I suspect that as with `bcryptjs` this has been pulled in by something I've added, let me remove it and re-test.

Reply:
@louislam - I've removed both and checked the docker container builds correctly and continues to function, we're all good so I've committed the changes to remove them.